src/factory/tests/HeaderPredicateFactory.spec.ts

```diff
@@ -446,6 +446,36 @@ describe('composedFromUri', () => {
             expect(predicate(measureHeaders.identifierBasedCompareArithmeticMeasure, context)).toBe(true);
         });
     });
+
+    describe('derived from AM', () => {
+        it('should match when derived PP from AM operand matches header uri', () => {
```

Comment:
`should match when derived PP from AM matches header uri` ?

Reply:
👍